src/app/auth/authentication.service.ts

```diff
@@ -189,10 +189,11 @@ export class AuthenticationService {
     let tokenUrl = this.apiUrl + 'token?for=https://github.com';
     const xhr = new XMLHttpRequest();
     xhr.open("delete", tokenUrl);
+    var that = this;
```

Comment:
`var` should be `let`
What do you need `that` for?

Reply:
It just to access the gitHubToken object from the inner function of the callback.  I've replaced it with a more specific variable, assuming that it will be accessed in order to update the token value.